app/helpers/articles_helper.rb

```diff
@@ -41,11 +41,12 @@ def should_show_updated_on?(article)
   end
 
   def should_show_crossposted_on?(article)
-    article.crossposted_at &&
+    article.canonical_url ||
+      (article.crossposted_at &&
```

Comment:
The parens aren't necessary for boolean logic, but without them Rubocop wants me to indent all the following lines by 2 more spaces. You can't always let the machines win.